doc/releases/release-notes-3.5.rst

```diff
@@ -361,6 +384,19 @@ Drivers and Sensors
 
 * WiFi
 
+  * Increased esp32 default network (TCP workq, RX and mgmt event) stack sizes to 2048 bytes.
+  * Added support for esp32c3_luatos_core board.
+  * Added support for esp32s3_luatos_core board.
+  * Added support for xiao_esp32s3 board.
```

Comment:
mmmm should just be listed in the Xtensa boards?

Reply:
These are related to wifi support in those boards. I can certainly remove all of them from this section.